### ISSUE TITLE
Add an easy way to plot the Solar System

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#300]
🎯 **Goal**

In the same way that there are example orbits, it would be nice to be able to plot the whole Solar System in an easy way, with appropriate colors and labels.

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by Juanlu001** [comment]
Some thoughts on the API taken from [the chat](https://riot.im/app/#/room/#poliastro:matrix.org): for instance, you can do

```
from poliastro.examples import molniya
plot(molniya)
```

but one cannot do `plot(solar_system)`, since that is made up of several Orbit objects.

On the other hand,

```
from poliastro.bodies import solar_system

frame = OrbitPlotter()
for planet in solar_system: 
    frame.plot(planet)
```

is not very straightforward to me.

Another option would be

```
from poliastro.plotting.helpers import plot_solar_system
plot_solar_system()
```

where plot_solar_system(epoch=None) will plot today's osculating orbits by default.

**Comment by shreyasbapat** [comment]
Is there any problem in taking this approach?
```
from poliastro.bodies import solar_system

frame = OrbitPlotter()
for planet in solar_system: 
    frame.plot(planet)
```

**Comment by Juanlu001** [comment]
My problem with the first approach is that `solar_system` would be a list of `Orbit` objects, while the rest of the objects in that module are `Body`es. The second looks cleaner to me.

**Comment by shreyasbapat** [comment]
Please Review the Pull Request #303

**Comment by Juanlu001** [comment]
Closed in #304
